airflow/www/views.py

```diff
@@ -1377,7 +1377,7 @@ def log(self, session=None):
         dag_id = request.args.get('dag_id')
         task_id = request.args.get('task_id')
         execution_date = request.args.get('execution_date')
-        dttm = timezone.parse(execution_date)
+        dttm = timezone.parse(execution_date) if execution_date else None
```

Comment:
isn't execution date deprecated and replaced with run id?

Reply:
Eventually we want to change this to use `run_id` instead so let’s not bother changing it. This is considered internal and `execution_date` is only a variable name.